ILSpy.AddIn/ILSpy.AddIn.csproj

```diff
@@ -1,10 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project>
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
-
-  <PropertyGroup>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
-  </PropertyGroup>
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
<Project>
  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

Reply:
This was the core change, and it build fine here...

Reply:
If there is no change left for part#1, splitting this does not really make sense...

Reply:
There are two expected changes for the PR

1. Using Microsoft.VSSDK.BuildTools instead of defining `VSToolsPath` manually.
2. Simplifying the definition on `DefineConstants`.

There should not be any changes to the `<Import>` elements until the next PR.